Framework/DataObjects/test/TimeSplitterTest.h

```diff
@@ -180,4 +183,64 @@ class TimeSplitterTest : public CxxTest::TestSuite {
     TS_ASSERT(!roi.empty());
     TS_ASSERT_EQUALS(roi.numBoundaries(), 4);
   }
+
+  void test_timeSplitterFromMatrixWorkspace() {
```

Comment:
Can you please please make the testing method name more transparent:

`test_timeSplitterFromMatrixWorkspace` ->  `test_timeSplitterFromMatrixWorkspaceAbsoluteTimes'